domain/identity/src/main/java/usecase/LogOutUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package usecase
+
+import javax.inject.Inject
+import repository.AuthenticationRepository
+
+class LogOutUseCase @Inject constructor(
+    private val authenticationRepository: AuthenticationRepository
+) {
+    suspend fun logout(){
```

Comment:
It will be better if this is a compact function
```suggestion
    suspend fun logout() = authenticationRepository.logout()
```